pyproject.toml

```diff
@@ -63,6 +63,7 @@ extend-select = [
 ]
 ignore = [
     "E731",  # Do not assign a lambda expression, use a def
+    "S108",
```

Comment:
Is this intentional? which rule is this?

Reply:
Yes, this is to avoid ruff to complain about the usage of `/tmp`